### PR TITLE
Create notifications table 

### PR DESCRIPTION
**Pull request by badekk**
## Create notifications table 
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## How has this been tested?

- [ ] Unit test(s)
- [ ] Integration test(s)
- [x] Tested manually

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I have checked my code and corrected any misspellings
- [ ] I have added frontend code and checked it for mobile & desktop

### CODE REVIEW

**Comment by mar-pan** [comment]
Task linked: [CU-862j9crcq Notifications table](https://app.clickup.com/t/862j9crcq)

File: src/backend/Auth/Actions/ManageUsersActions/ManageUsersGetSingle/ManageUsersGetSingleAction.cs [hunk 1]
```diff
@@ -38,6 +39,7 @@ public async Task<OkObjectResult> Execute(string id)
                  .Include(c => c.Roles)!
                      .ThenInclude(c => c.Role)
                  .Include(c => c.Warehouse)
+                 .Where(c => c.Id != UserSeeder.ServiceUser.Id)
```
Comment (h4570) [inline src/backend/Auth/Actions/ManageUsersActions/ManageUsersGetSingle/ManageUsersGetSingleAction.cs:42]: Już tu mogłeś sobie darować, bo czysto hipotetycznie nigdy się to nie stanie.
Ale może zostać

File: src/backend/Core/Database/Models/Core/Notification.cs [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Core.Database.Models.Core
+{
+    public class Notification : DbEntity, IEntityTypeConfiguration<Notification>
+    {
+        public long UserId { get; set; }
+        public User? User { get; set; }
+        public string? Message { get; set; }
```
Comment (h4570) [inline src/backend/Core/Database/Models/Core/Notification.cs:13]: Message jest not null, zatem `string`

File: src/backend/Core/Database/Models/Core/Notification.cs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+﻿using Core.Database.Base;
+using Core.Database.Config.Models.Core;
+using Core.Database.Models.Auth;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Core.Database.Models.Core
+{
+    public class Notification : DbEntity, IEntityTypeConfiguration<Notification>
```
Comment (h4570) [inline src/backend/Core/Database/Models/Core/Notification.cs:9]: Do `AppDbContext` chyba tabeli nie dodałeś.

**Review by h4570 — CHANGES_REQUESTED** [review]

File: src/backend/Auth/Actions/ManageUsersActions/ManageUsersGetSingle/ManageUsersGetSingleAction.cs (hunk 1 above)
Comment (badekk) [reply src/backend/Auth/Actions/ManageUsersActions/ManageUsersGetSingle/ManageUsersGetSingleAction.cs:42]: fixed

File: src/backend/Core/Database/Models/Core/Notification.cs (hunk 2 above)
Comment (badekk) [reply src/backend/Core/Database/Models/Core/Notification.cs:13]: fixed

File: src/backend/Core/Database/Models/Core/Notification.cs (hunk 3 above)
Comment (badekk) [reply src/backend/Core/Database/Models/Core/Notification.cs:9]: fixed

**Pull request merged**